source/client/options_impl.cc

```diff
@@ -265,7 +265,14 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       "Remote gRPC source that will deliver to-be-replayed traffic. Each worker will separately "
       "connect to this source. For example grpc://127.0.0.1:8443/.",
       false, "", "uri format", cmd);
-
+  TCLAP::ValueArg<std::string> request_source_plugin_config(
+      "", "request-source-plugin-config",
+      "Request Source plugin configuration in json or compact yaml. "
+      "Mutually exclusive with --request-source. Example (json): "
```

Comment:
Is the mutual exclusivity enforced? Or will one option be prioritized over the other when both are configured?

Reply:
The mutually exclusivity should be enforced. I added a validation in the constructor.